drive-stack/net-detec/BUCK

```diff
@@ -0,0 +1,36 @@
+rust_library(
+    name = "lib",
+    crate = "net_detec",
+    srcs = glob(["src/**/*.rs"], exclude=["main.rs"]),
+    edition = "2024",
+    deps = [
+        "//third_party:mdns-sd",
+        "//third_party:thiserror",
+        "//third_party:if-addrs",
+    ],
+    target_compatible_with = [
+        "prelude//os/constraints:linux",
```

Comment:
is there actually anything that makes this only compatible with linux? if not, remove